spacialgaze-plugins/cards.js

```diff
@@ -0,0 +1,908 @@
+'use strict';
+/********************
+ * Cards
+ * Avalon's Card System: Credit to Naten, nineage, fender, and everyone who added cards/
+ * All cards should be retrieved here http://www.pokemon.com/us/pokemon-tcg/pokemon-cards/
+ * Cards are organized alphabetically and use a point system
+ * publicids are dex numbers and any unique identifiers (if they're not a Pokemon, do a shorthand version of the card name)
+ * Dex Number, (for multiple pokemon: DEX[lowercase letter, a, b, c, d])
+ ********************/
+const uuid = require('uuid');
+const cards = require('../config/card-data.js');
+let color = require('../config/color');
```

Comment:
I do test **always** But sometimes I may get screwed and forget to test all the stuff